starter/.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
 ./node_modules/.bin/lint-staged && npm run docs:coverage
```

Comment:
Could you adapt this as following

```
lint-staged && npm run docs:coverage
```

You can check the PR on ngx-form-errors: https://github.com/NationalBankBelgium/ngx-form-errors/pull/51